src/sound_openal.cpp

```diff
@@ -0,0 +1,865 @@
+#ifdef APPLE
```

Comment:
This needs to go after Config.hpp is included, otherwise APPLE has not yet had a chance to be defined.

This file is also missing the preamble...